code/modules/reagents/Chemistry-Reagents.dm

```diff
@@ -6302,16 +6302,23 @@
 	dupeable = FALSE
 
 /datum/reagent/ethanol/scientists_serendipity/handle_special_behavior(var/obj/item/weapon/reagent_containers/food/drinks/drinkingglass/D)
-	if(volume<10)
+	if(volume < 10)
 		glass_icon_state = "scientists_surprise"
 		glass_name = "\improper Scientist's Surprise"
 		glass_desc = "There is as yet insufficient data for a meaningful answer."
-	else
+
+	if(volume > 10) && (volume < 50)
```

Comment:
The `(volume > 10) && (volume < 50)` would have to be wrapped in a pair of parentheses, but I would just recommend doing
```suggestion
	else if(volume < 50)
```
instead.